### ISSUE TITLE
ext/SDBM_File/t/corrupt.t: running with warnings flags probably typo

### ISSUE THREAD

**Issue by jkeenan** [Perl/perl5#17970]
In commit 7f4a9bc72716574716b99e45c11129541a8fbeb7 in Nov 2018, @tonycoz added this file to the core distribution:
```
diff --git a/ext/SDBM_File/t/corrupt.t b/ext/SDBM_File/t/corrupt.t
new file mode 100644
index 0000000000..b4e41aab64
--- /dev/null
+++ b/ext/SDBM_File/t/corrupt.t
@@ -0,0 +1,86 @@
+#!./perl
+use strict;
+use Test::More;
+use MIME::Base64;
+use File::Temp 'tempfile';
+use SDBM_File;
+use Fcntl qw(O_RDWR);
+use Errno qw(EINVAL);
+
+my ($dirfh, $dirname) = tempfile(UNLINK => 1);
+my ($pagfh, $pagname) = tempfile(UNLINK => 1);
+close $dirfh;
+close pagefh;
...
```
Notice that the last statement shown above is **not** `close $pagfh` but `close pagefh` instead.  I believe this to have been a typo.  If the file is run with warnings enabled -- as we are doing in Perl 7 -- this statement is immediately flagged.
```
$ perl -w -Iext/SDBM_File ext/SDBM_File/t/corrupt.t 
Unquoted string "pagefh" may clash with future reserved word at ext/SDBM_File/t/corrupt.t line 19.
Name "main::pagefh" used only once: possible typo at ext/SDBM_File/t/corrupt.t line 19.
ok 1 - 132147: tied successfully
...
```

Was this indeed a spelling error?

Thank you very much.
Jim Keenan

**Comment by atoomic** [comment]
IMO this is also a typo and should be `close $pagefh` using strict and warnings would have exposed this
we should fix it in p7 branch and submit a PR to blead too

**Comment by tonycoz** [comment]
Looks like a typo to me.

**Comment by jkeenan** [comment]
Fixed in 7d036d4038.  Closing.

Thank you very much.
Jim Keenan
